orc8r/protos/bootstrapper.proto

```diff
@@ -71,3 +73,83 @@ service Bootstrapper {
   // Returns signed certificate.
   rpc RequestSign (Response) returns (Certificate) {}
 }
+
+// --------------------------------------------------------------------------
+// Protos for CloudRegistration and Registration servicer
+// --------------------------------------------------------------------------
```

Comment:
This file is organized a little strangely with the additions. A typical pattern we've followed for `.proto` files is to to group the messages under the corresponding service definition.

Reply:
They are pretty connected though, and I think it might be easier to understand if they next to each other?